R/use_lintr.R

```diff
@@ -0,0 +1,45 @@
+#' Use lintr in your project
+#'
+#' Create a minimal lintr config file as a starting point for customization
+#'
+#' @param path Path to project root, where a `.lintr` file should be created.
+#' If it already exists, an error will be thrown.
+#' @param type What kind of configuration to create?
+#'
+#'  * `minimal` creates a minimal lintr config, based off the default linters ([linters_with_defaults()]).
+#'  * `full` creates a lintr config using all available linters using [linters_with_tags()].
```

Comment:
"using" twice... "all available linters in [linters_with_tags()]"?

Reply:
maybe via?

Reply:
works for me